src/l500/l500-private.h

```diff
@@ -333,14 +333,26 @@ namespace librealsense
         {
             orientation orient;
             resolutions_depth resolution;
+
+            size_t get_expected_size(const std::vector<uint8_t> & raw_data_vec) const 
```

Comment:
I don't understand this:
We already know our type, which is already cast from the raw data... so why do we need to pass the raw_data_vec in here?

Reply:
I wanted to make sure that the function will get real input.
I am using a field of this struct as input, the only way I know for sure that the input is real is if I get the raw one.
if I use the struct field directly the result will be garbage if it was not set or pointed to real input. 

any other suggestion?

Reply:
One way is to make the function static.
But I don't think it's needed -- you already did the casting to this type before calling this function, so the danger is already there.
Just comment it well.

Reply:
OK, Will comment.